tests/compute/subresources/reset.go

```diff
@@ -48,25 +48,21 @@ var _ = Describe(compute.SIG("Reset subresource", func() {
 			By("Checking that the VirtualMachineInstance console has expected output")
 			Expect(console.LoginToAlpine(vmi)).To(Succeed())
 
-			By("Create a file that is not expected to survive the reset request")
-			err := console.SafeExpectBatch(vmi, []expect.Batcher{
-				&expect.BSnd{S: "touch /tmp/non-persistent-file\n"},
-				&expect.BExp{R: console.PromptExpression},
-				&expect.BSnd{S: console.EchoLastReturnValue},
-				&expect.BExp{R: console.ShellSuccess},
-			}, 120)
+			By("Store boot time pre and post reset")
+			cmd := "cat /proc/stat | grep btime"
+			bTimePreReset, err := console.RunCommandAndStoreOutput(vmi, cmd, time.Second*30)
 			Expect(err).ToNot(HaveOccurred())
 
 			err = kubevirt.Client().VirtualMachineInstance(testsuite.GetTestNamespace(vmi)).Reset(context.Background(), vmi.Name)
 			Expect(err).ToNot(HaveOccurred())
 
 			Expect(console.LoginToAlpine(vmi)).To(Succeed())
-			err = console.SafeExpectBatch(vmi, []expect.Batcher{
-				&expect.BSnd{S: "ls /tmp/non-persistent-file\n"},
-				&expect.BExp{R: `non-persistent-file: No such file or director`},
-			}, 20)
+			bTimePostReset, err := console.RunCommandAndStoreOutput(vmi, cmd, time.Second*30)
 			Expect(err).ToNot(HaveOccurred())
 
+			By("Check the pre and post reset boot times are different")
+			Expect(bTimePreReset).ToNot(Equal(bTimePostReset))
```

Comment:
Sorry to chime in late, but I think we should ensure that this string is not empty, so that if there's a silent error we won't be passing as a false positive.

I've created a tiny follow-up for that: https://github.com/kubevirt/kubevirt/pull/14321.